### DISCUSSION TITLE
internal HDD [Samba NAS] not showing up on LG TV

### DISCUSSION THREAD

**Discussion by ufacek** [dianlight/hassio-addons#282] [Q&A]
I have an ext4 disk on which I have recorded video files, I would like to share this disk to my LG TV, but the TV does not see this disk.
Windows 11 sees this disk, can you tell me where I am wrong?

**Comment by thcuba** [comment]
If win 11 see your hard drive and not the TV the solution Is on the TV. 
Check correct url
